201-decrypt-running-linux-vm/azuredeploy.json

```diff
@@ -0,0 +1,75 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "vmName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the virtual machine"
+      }
+    },
+    "volumeType": {
+      "type": "string",
+      "defaultValue": "Data",
+      "allowedValues": [
+        "Data"
+      ],
+      "metadata": {
+        "description": "Decryption is supported only on data drives for Linux VMs."
+      }
+    },
+    "sequenceVersion": {
+      "type": "string",
+      "defaultValue": "1.0",
+      "metadata": {
+        "description": "Pass in an unique value like a GUID everytime the operation needs to be force run"
+      }
+    }
+  },
+  "variables": {
+    "extensionName": "AzureDiskEncryptionForLinux",
+    "extensionVersion": "0.1",
+    "encryptionOperation": "DisableEncryption",
+    "updateEncryptionSettingsUrl": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/201-decrypt-running-linux-vm/updateEncryptionSettings.json"
+  },
+  "resources": [
+    {
+      "type": "Microsoft.Compute/virtualMachines/extensions",
+      "name": "[concat(parameters('vmName'),'/', variables('extensionName'))]",
+      "apiVersion": "2016-03-30",
```

Comment:
The latest apiVersion for virtual machine extension resources is "2015-06-15". Please use that value here.